utility/doc/TRUSTLESS_RELAY_VALIDATION.md

```diff
@@ -0,0 +1,73 @@
+# Validation of Trustless Relays
+
+- [A. Client-side Relay Validation](#a-client-side-relay-validation)
```

Comment:
Since you didn't use `<!-- omit from toc -->` in the title, I'm guessing you did this by hand?

Highly recommend using this extension in the future: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one